kml/serdes_gpx.cpp

```diff
@@ -540,7 +543,17 @@ void SaveTrackData(Writer & writer, TrackData const & trackData)
   writer << "<trk>\n";
   auto name = GetDefaultLanguage(trackData.m_name);
   if (name.has_value())
-    writer << kIndent2 << "<name>" << name.value() << "</name>\n";
+  {
+    writer << kIndent2 << "<name>";
+    SaveStringWithCDATA(writer, name.value());
```

Comment:
nit:
- I'd prefer to have ```if (name) Save(*name)``` notation
- We can return ```std::string const * GetDefaultLanguage(...)``` instead of ```optional<string>```

Reply:
If such refactoring is beneficial I can have a look on it, but in this case it should be done for all occurrences in kml/gpx parsers to be consistent.

Reply:
Yes, in all occurrences. Up-to-you, the PR is ok to merge.

Reply:
@vng I reworked GetDefaultLanguage as you suggested, moved it to common and re-used in kml serdes where applicable. Please have a look and let me know if something else is required for this rework.